ports/raspberrypi/common-hal/usb_host/Port.c

```diff
@@ -57,6 +58,7 @@ static void __not_in_flash_func(core1_main)(void) {
     SysTick->CTRL = SysTick_CTRL_CLKSOURCE_Msk |  // Processor clock.
         SysTick_CTRL_ENABLE_Msk;
 
+#if 0
```

Comment:
@tannewt This PR get USB host running but this MPU code need to be commented out. It is probably due to some code not entirely in RAM (or still ROM access e.g const data). I guess this already made some progress for you to work on.